jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java

```diff
@@ -12,7 +12,7 @@
 import selenium.site.sections.LoginForm;
 
 @JSite("https://jdi-testing.github.io/jdi-light/")
-public class SiteJdi {
+public class StaticSiteJdi {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'StaticSiteJdi' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

Reply:
it does